### PR TITLE
FU-804 Move Pagination to top of the page (all grid views)

### PR DESCRIPTION
**Pull request by ni-frolov**

### CODE REVIEW

File: dist/grid.html [hunk 1]
```diff
@@ -202,6 +202,15 @@
 
     <require from="./pager"></require>
 
+    <div class="grid-container table  ${class} grid-container-outer"  if.bind="lastVisibleItem > 10">
+        <div class="grid-footer-container " if.bind="pageable && !loading && count > pageSize">
+            <pager page-size.bind="pageSize " num-to-show.bind="pagerSize " pager.ref="pager " on-page-changed.call="pageChanged($event)
```
Comment (imressed) [inline dist/grid.html]: what is pager?

File: dist/grid.js [hunk 2]
```diff
@@ -281,6 +275,17 @@ System.register(['aurelia-framework', './grid-column', './grid-columns-expander'
                     if (removeResponse instanceof Promise) {
                         removeResponse.then(function () { return _this.viewSlot.add(view); });
                     }
+                    if (this.expanderAttrs) {
+                        //compile expander container one time and place inside grid, then it's inserted and shown below active row
+                        var innerDiv = document.createElement("div");
+                        innerDiv.setAttribute("id", "expander-container");
+                        innerDiv.setAttribute("class", "inner-container");
+                        innerDiv.setAttribute("style", "display:none");
+                        innerDiv.innerHTML = '<compose view-model="' + this.expanderAttrs.viewModel + '"model.bind="' + this.expanderAttrs.model + '"  ></compose>';
```
Comment (imressed) [inline dist/grid.js]: why this weird code present here?

Comment (ni-frolov) [reply dist/grid.js]: it's after build, it's not my changes

File: dist/grid.html (hunk 1 above)
Comment (ni-frolov) [reply dist/grid.html]: it's a pagination

File: dist/grid.js (hunk 2 above)
Comment (ni-frolov) [reply dist/grid.js]: my changes in src/grid.html only

**Pull request merged**